cypress/integration/home.spec.ts

```diff
@@ -2,3 +2,48 @@ it('should redirect to home page', () => {
   cy.visit('/');
   cy.location('pathname').should('eq', '/');
 });
+
+it('should create a new task', () => {
+  cy.get('[data-testid="add-task-input"]').type('Created by Cypress');
```

Comment:
I suggest adding this selector into a function.

```suggestion
  getTaskInput().type('Created by Cypress');
```

Reply:
Resolved on 10f2a13bb740a5523c33457995b2091cf9774377.